frontend/src/restful/restful.js

```diff
@@ -5,13 +5,17 @@ const loginOrRegister = () => {
   window.location = `/users/identify?from=${window.location.href}`;
 };
 
+const jsonResponseHelper = async (response) => {
```

Comment:
Two things bother me about this change. One is the name of the method. What is this method doing for the calling method? I can't really tell.

The second issue is the duplicated "=== 400". That suggests to me that there is something missing. I'd start by duplicating the if statement, so there's one whose condition is === 200 and another whose condition is === 400 (case statement would be okay too). Then I'd tidy from there.

Reply:
Never mind. I didn't bother to look where you were going. I'm okay with this sequence.